bbc/simorgh
# WSTEAM1-705: Links to posts on live page (SPIKE)

Resolves JIRA [[WSTEAM1-705]](https://jira.dev.bbc.co.uk/browse/WSTEAM1-705)

Overall changes
======
Spike to explore Linking to Posts/ Linkable Key Points. Based on PS implementation. See the dropbox paper linked to the ticket.

Code changes
======
- Hard code in some test links
- Passes post param into 'getPageData' function
- Add 'scrollToLatest' hook which uses 'scroll-into-view' helper (doesn't work with links that end with `#post`)
- Render 'BackToLatest' link above selected post
- Use `#post` to direct user focus to the 'BackToLatest' link
- Hard code in stream 1st page link
- Add a Stream Provider (not sure I'm using this to it's full extent).
- Modify NextJs app set up so to connect to local BFF
- Add some hacky 'copyToClipboard'/ Share helpers.

Testing
======
1. Pull down this PR and the correlating BFF PR
2. Connect and run locally
3. Go to http://localhost:7081/pidgin/live/c07zr0zwjnnt
4. See links on the page.


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
